src/App.tsx

```diff
@@ -1,61 +1,111 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { FC, useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+
+const todos = todosFromServer.map((todo) => {
+  return {
+    ...todo,
+    user: usersFromServer.find((user) => user.id === todo.userId) || null,
+  };
+});
```

Comment:
Why do you need return here?
```suggestion
const todos = todosFromServer.map((todo) => ({
    ...todo,
    user: usersFromServer.find((user) => user.id === todo.userId) || null,
  };
));
```